reference/shipping.v2.yml

```diff
@@ -1377,7 +1393,8 @@ paths:
                         "shipping_cost": 0
                     }
                 ]
-            }
+            },
+            "channels": [1, 3]
```

Comment:
🍻 I think it'd actually be better to avoid updating most of our existing examples.

At the moment channel management isn't an overly common use case. In this specific example, the core example isn't speaking to channels.

> This example sets free shipping above a certain order total

Hence I think it's a distraction to include the channels field here, and probably in all of the existing examples.

If we reconsider this from the ground up, focusing on the specific use cases where we are creating a simple shipping method for a channel, then it might be better. That is, as part of the description specifying why the `channels` field has been used and what it's effect is.

Reply:
Thanks Brett, you're absolutely right. With the post and put examples I've introduced this new one dedicated to channels.

This particular line though falls under the get, and channels field will be present in every call, so I'm keeping the channels in this area.

This looks clearer now.🍻🙏